safenode/src/protocol/types/wallet/network_store.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright 2023 MaidSafe.net limited.
```

Comment:
this is actually an empty file ?

maybe at least put some pseudo code or TODO comments in?